dev/AnimatedIcon/AnimatedIcon.vcxitems

```diff
@@ -18,32 +18,38 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedIcon.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedIconTestHooks.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_01_Checkmark.h" />
-    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_02_UpDown_Dropdown.h" />
-    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_07_Settings.h" />
-    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_08_Back.h" />
-    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_09_Hamburger.h" />
-    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_10_Search.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_02_Hamburger.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_03_Back.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_04_Settings.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_05_Search.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_06_DownChevron_DropDown.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_07_DownChevron_NavView.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AnimatedVisuals\Controls_08_SideChevron_TreeView.h" />
```

Comment:
I know it's too late to raise this question, but why did we opt for a number prefix for the visuals? Do the numbers convey any meaning?

Reply:
These are placeholder names and will change in the near future. If you have naming suggestions its not to late.

Reply:
No meaning. These are temporary until we pick reasonable names with API review.

Reply:
Ah I see, thanks.